exporters/otlp/internal/otlpconfig/envconfig.go

```diff
@@ -145,6 +145,18 @@ func (e *EnvOptionsReader) GetOptionsFromEnv() []GenericOption {
 	return opts
 }
 
+func endpointOptions(endpoint string, endpointOption func(endpoint string) GenericOption, insecureOption func(insecure bool) GenericOption) []GenericOption {
+	var endpointOptions []GenericOption
+
+	endpointOptions = append(endpointOptions, insecureOption(strings.HasPrefix(endpoint, "http://")))
```

Comment:
bug: `HTTP://` prefix will be considered as "safe"
would be good to add a test case for it

Reply:
I added support for uppercase scheme.

Reply:
This comment can be resolved